.circleci/config.yml

```diff
@@ -82,18 +98,19 @@ workflows:
             branches:
               ignore:
                 - /.*/
-  weekly:
+  tests:
+    jobs:
+      - run-coverage
+      - run-d8-codeception
+      - run-d9-codeception
+  # Re-test every sunday in case this code becomes stale.
+  sundays:
     jobs:
       - run-coverage
-#      - run-codeception
     triggers:
       - schedule:
           cron: "0 0 * * 0"
           filters:
             branches:
               only:
-                - 8.x-2.x
-  tests:
-    jobs:
-      - run-coverage
-#      - run-codeception
+                - 8.x-1.x
```

Comment:
Shouldn't this be 2.x?

Reply:
yeah. i think that was just search and replace mistake